src/components/ListItem.jsx

```diff
@@ -12,12 +13,10 @@ export function ListItem({ name, listToken, itemId, data }) {
 
 	useEffect(function unCheckAfterTime() {
 		if (data.dateLastPurchased !== null) {
-			const now = Date.now();
-			const oneDayInSeconds = 24 * 60 * 60;
-			const newDateLastPurchased = data.dateLastPurchased.seconds;
-			const limit = (newDateLastPurchased + oneDayInSeconds) * 1000;
+			const yesterday = getFutureDate(-1);
+			const dateLastPurchased = new Date(data.dateLastPurchased.toDate());
 
```

Comment:
refactor for using less variables and utilizing the already created getFutureDate() function.  Plus we don't have to ignore the nanoseconds portion of the timestamp.

yesterday: getFutureDate with argument '-1' to get the date 24hrs before now.
dateLastPurchased: uses new Date() object with the argument of a timestamp converted from the firebase document (that's what the .toDate() method does).

Line 19 - if yesterday is more recent (greater than) dateLastPurchased is equivalent to now being later than 1 day after dateLastPurchased.

Reply:
I'm not sure what you're referring to with "line 19". There is just a close brace?

It might be easier to just click the plus button next to the line you're referring and make a new comment?

Reply:
Would be more succinct to just put a recommended commit, see above.

Reply:
In anycase, I've gone off on a tangent, I think this comment needs to be resolved as it's talking about a code that's already been changed.